pkgs/data/fonts/material-symbols/default.nix

```diff
@@ -28,12 +29,17 @@ stdenvNoCC.mkDerivation {
     runHook postInstall
   '';
 
+  passthru.updateScript = unstableGitUpdater { };
+
   meta = with lib; {
```

Comment:
Remove `with lib;` from here, and add `lib.` to expressions that need it in the subset.